MessageBox.cpp

```diff
@@ -75,7 +76,7 @@ void MessageBox::about() {
                     << "Ukrainian: Ihor Hordiichuk" // <igor_ck@outlook.com>
                     << "Standreas and the entire LXQt weblate team"
                     ;
-    QString aboutString = "<h1>Phototonic 3.0 β²</h1>" // + QString(VERSION)
+    QString aboutString = "<h1>Phototonic " + QString(phototonic::project_version) + " β²</h1>" // + QString(VERSION)
```

Comment:
See main.cpp, "QCoreApplication::setApplicationVersion(VERSION);"
For the GUI, I'd prefer a manual string, certainly over some git hash (see the commented tail) - even if that requires, well, manual labor.